src/hooks/loadables/useLoadBalances.ts

```diff
@@ -27,10 +27,16 @@ export const useTokenListSetting = (): boolean | undefined => {
   return isTrustedTokenList
 }
 
-const mergeBalances = (balance1: SafeBalanceResponse, balance2: SafeBalanceResponse): SafeBalanceResponse => {
+const mergeBalances = (cgw: SafeBalanceResponse, sn: SafeBalanceResponse): SafeBalanceResponse => {
+  // Create a Map using token addresses as keys
+  const uniqueBalances = new Map(
+    // Process SafeNet items first so they take precedence
+    [...sn.items, ...cgw.items].map((item) => [item.tokenInfo.address, item]),
```

Comment:
I think the cgw items will overwrite the SafeNet items if SafeNet items are processed first
e.g.
```ts
new Map([[1,"one"], [1, "two"]])
 => Map(1) {1 => 'two'}
```

Reply:
Strange because it worked when testing, I'll look into it again

Reply:
You were right sir, thanks for the catch 👍